src/tests/contracts/GameItems/abi.ts

```diff
@@ -0,0 +1,72 @@
+export const ABI: any = [
```

Comment:
Delete this file is already in [HRC1155](https://github.com/blockcoders/harmony-marketplace-sdk/blob/main/src/tests/contracts/HRC1155/abi.ts)

Reply:
Copy!